README.md

```diff
@@ -12,15 +12,48 @@ The production bot runs on the c0d3 CapRover server and can be communicated with
 
 ## Development - Getting Started
 
-\* First you need to register a discord application to get the `DISCORD_TOKEN` that c0d3r will use to forward your commands to your discord server. A guide for that setup can be found on the wiki page here: [Registering a Discord Bot](https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot)
+First you need to register a discord application to get the `BOT_TOKEN` that c0d3r will use to forward your commands to your discord server. A guide for that setup can be found on the wiki page here: [Registering a Discord Bot](https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot)
 
-- ## Using yarn
+## Setting the .env variables
+
+Copy and rename `example.env` to `.env` and fill the required variables below
+
+`DISCORD_TOKEN`: to find your Discord token, you can [follow these instructions](https://www.androidauthority.com/get-discord-token-3149920/).
```

Comment:
The discord token this article is about is unrelated to the one needed for the bot. The link above, https://github.com/garageScript/c0d3r/wiki/Registering-a-Discord-Bot#registering-a-discord-bot is how we can get the DISCORD_TOKEN